feature/experimental/gribi/otg_tests/encap_decap_scale/README.md

```diff
@@ -0,0 +1,312 @@
+                            sequence-id: 10
+                            ipv4 {
+                                protocol: 41
+                                source-address: "ipv4_outer_src_222"
+                            }
+                            action {
+                                decap-network-instance: "DECAP_TE_VRF"
+                                post-network-instance: "DEFAULT"
+                                decap-fallback-network-instance: "TE_VRF_222"
+                            }
+                        }
+                        rule {
+                            sequence-id: 11
+                            ipv4 {
+                                protocol: 4
+                                source-address: "ipv4_outer_src_111"
+                            }
+                            action {
+                                decap-network-instance: "DECAP_TE_VRF"
+                                post-network-instance: "DEFAULT"
+                                decap-fallback-network-instance: "TE_VRF_111"
+                            }
+                        }
+                        rule {
+                            sequence-id: 12
+                            ipv4 {
+                                protocol: 41
+                                source-address: "ipv4_outer_src_111"
+                            }
+                            action {
+                                decap-network-instance: "DECAP_TE_VRF"
+                                post-network-instance: "DEFAULT"
+                                decap-fallback-network-instance: "TE_VRF_111"
+                            }
+                        }
+                        rule {
+                            sequence-id: 13
+                            action {
+                                network-instance: "DEFAULT"
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+## Procedure
+
+1.  via gRIBI installs the following AFT entries:
+    *   Add 4 VRFs for encapsulations: `ENCAP_TE_VRF_A`, `ENCAP_TE_VRF_B`, `ENCAP_TE_VRF_C` and `ENCAP_TE_VRF_D`.
+    *   Inject 5000 IPv4Entry-ies and 5000 IPv6Entry-ies to each of the 4 encap VRFs.
+    *   The entries in the encap VRFs should point to NextHopGroups in the `DEFAULT` VRF. Inject 200 such NextHopGroups in the DEFAULT VRF.
+    *   Each NextHopGroup should have 8 NextHops where each NextHop points to a tunnel in the `TE_VRF_111`. In addition, the weights specified in the NextHopGroup should be co-prime and the sum of the weights should be 16.
+    *   Add 1 VRF for decapsulation, `DECAP_TE_VRF`.
+    *   Inject `48` entries in the DECAP_TE_VRF where the entries have a mix of prefix lengths /22, /24, /26, and /28.
+2.  Send the following packets to DUT-1
+
+    ```
+    * inner_src: `ipv4_inner_src`
+    * inner_dst: `ipv4_inner_encap_match`
+    * dscp: `dscp_encap_a`
```

Comment:
For all of these packet templates, should this line be `inner_dscp` and the next one `outer_dscp`, to be more clear?

Reply:
If it below Inner_src/Dst means inner header dscp bit. ?